src/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const saveLogin = state => {
+  try {
+    const serializedstate = JSON.stringify({ authentication: state });
+    localStorage.setItem("authentication", serializedstate);
```

Comment:
This will work, but be aware that ios can choose to clear localstorage when the device needs space:

https://cordova.apache.org/docs/en/latest/cordova/storage/storage.html#disadvantages

a solution could be: https://github.com/TheCocoaProject/cordova-plugin-nativestorage